src/bootman/bootman.c

```diff
@@ -456,20 +439,17 @@ int mount_boot(BootManager *self, char **boot_directory)
 
         /* Determine root device */
         root_base = self->sysconfig->boot_device;
-        if (!root_base) {
-                LOG_FATAL("Cannot determine boot device");
-                goto out;
-        }
+        CHECK_FATAL_GOTO(!root_base, out, "Cannot determine boot device");
 
         abs_bootdir = cbm_system_get_mountpoint_for_device(root_base);
 
         if (abs_bootdir) {
                 LOG_DEBUG("Boot device already mounted at %s", abs_bootdir);
+
                 /* User has already mounted the ESP somewhere else, use that */
-                if (!boot_manager_set_boot_dir(self, abs_bootdir)) {
-                        LOG_FATAL("Cannot initialise with premounted ESP");
-                        goto out;
-                }
+                CHECK_FATAL_GOTO(!boot_manager_set_boot_dir(self, abs_bootdir), out,
+                                 "Cannot initialise with premounted ESP");
```

Comment:
Heh not your fault but may as well fix initialize spelling X(.

Reply:
done